examples/chip-tool/templates/commands.zapt

```diff
@@ -18,16 +21,52 @@
 #include <zap-generated/CHIPClientCallbacks.h>
 #include <zap-generated/CHIPClusters.h>
 
-// Value logging functions.  The non-generated ones depend on the
+#define SKIP_SPACE(iter)                                                                                                           \
```

Comment:
could we make this a function instead of a macro? 

I am thinking that functions (even if templated) are generally easier to read and maintain than multi-line macros.

Additionally, I expect template file contents to always depend on template inputs. This does not - it is a "helper/utility macro" so it should go in an appropriately named header (maybe struct-parsing or input-parsing or similar).